package.json

```diff
@@ -136,6 +135,7 @@
     "ora": "4.0.5",
     "prettier": "2.0.5",
     "protractor": "5.4.2",
+    "rollup-plugin-copy": "3.3.0",
```

Comment:
rollup-plugin-copy-asset doesn't allow me to move the proto files to dist/lite/src/proto